src/Svg.tsx

```diff
@@ -1,38 +1,39 @@
-// @flow
-import * as React from "react"
+import * as React from "react";
 
-import uid from "./uid"
-import type { Props as HolderProps } from "./Holder"
+import { IProps } from "./Holder";
 
-export type SvgProps = {
-  children?: React.ChildrenArray<*>
-} & HolderProps
+import uid from "./uid";
+
+export interface ISvgProps extends IProps {
+  children?: React.ReactNode;
+}
 
 export default ({
+  rtl,
+  speed,
+  style,
+  width,
+  height,
   animate,
   ariaLabel,
   children,
   className,
-  height,
-  preserveAspectRatio,
+  uniquekey,
   primaryColor,
   primaryOpacity,
-  rtl,
   secondaryColor,
   secondaryOpacity,
-  speed,
-  style,
-  uniquekey,
-  width,
+  preserveAspectRatio,
+  // tslint:disable-next-line:trailing-comma
```

Comment:
Is it really necessary? I think with the suggestion in the `tslint` that I did, it'll be solved

Reply:
done